internal/service/outposts/outpost_assets_data_source_test.go

```diff
@@ -37,3 +43,18 @@ data "aws_outposts_assets" "test" {
 
 `
 }
+
+func testAccOutpostAssetsDataSourceConfig_statusFilter() string {
+	return `
+data "aws_outposts_outposts" "test" {}
+
+data "aws_outposts_assets" "source" {
+  arn = tolist(data.aws_outposts_outposts.test.arns)[0]
+}
+
+data "aws_outposts_assets" "test" {
+	arn = data.aws_outposts_assets.source.arn
+	status_id_filter = ["ACTIVE"]
+  }
```

Comment:
Styling change to `terraform fmt` the code block

```suggestion
data "aws_outposts_assets" "test" {
  arn = data.aws_outposts_assets.source.arn
  status_id_filter = ["ACTIVE"]
}